src/seq.js

```diff
@@ -0,0 +1,27 @@
+import { curry, map, tap } from 'ramda';
+
+/**
+ * Runs the given list of functions in order with the supplied object, then returns the object.
+ * Also known as the normal order sequencing combinator.
+ *
+ * Acts as a transducer if a transformer is given as second parameter.
+ *
+ * @func seq
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
+ * @category Function
+ * @sig [(a -> *), (a -> *), ...] -> a -> a
+ * @param {Array} fns The list of functions to call in order with `x` whose return values will be
+ *        thrown away
+ * @param {*} x
+ * @return {*} `x`
+ * @see {@link http://ramdajs.com/docs/#tap|tap}
+ * @see {@link http://www.cs.rpi.edu/academics/courses/spring11/proglang/handouts/lambda-calculus-chapter.pdf|sequencing combinator detailed}
```

Comment:
Lets bee consistent with `Y` combinator link label and let's say `sequencing combinator explained`

https://github.com/char0n/ramda-adjunct/blob/9e2dcf1071bc570c17f24c8645be447ce1c1a55e/src/Y.js#L15